enterprise/backend/src/metabase_enterprise/metabot_v3/tools/create_dashboard_subscription.clj

```diff
@@ -12,7 +12,7 @@
   (if (int? dashboard-id)
     (let [dashboard (-> (t2/select-one :model/Dashboard :id dashboard-id)
                         (t2/hydrate [:dashcards :card]))
-          cards (for [{:keys [id card]} (sort-by (juxt :tab :row :col) (:dashcards dashboard))
+          cards (for [{:keys [id card]} (:dashcards dashboard)
```

Comment:
Ordering by `:tab` has never worked (there is no such thing in the DB, it's front end only). I don't think this ordering plays any role, so I've removed it.

Reply:
If it does turn out to be necessary, I believe you could hydrate `:tabs` on the dashboard, and each tab would have its own cards. But yeah, I don't think it matters here.